### PR TITLE
updated README to include Bower installation package name

### PR DESCRIPTION
**Pull request by Henryvw**
Just thought it would be nice for users looking to install it to make this fact explicit in the README without having to search Bower's Package Directory :-)

### CODE REVIEW

**Comment by jeffcarp** [comment]
Hi @Henryvw, thanks for the PR! The README definitely needs an installation section, thanks for bringing this up. However, I think it should match the style of our other repos - take the [Downloads section of card-validator](https://github.com/braintree/card-validator#download), for example. Also, it should mention both npm and bower.

**Comment by Henryvw** [comment]
Hi @jeffcarp , Done! thanks for the suggestion. You are very welcome. :-) Cheers

File: README.md [hunk 1]
```diff
@@ -1,10 +1,25 @@
-Credit Card Type [![Build Status](https://travis-ci.org/braintree/credit-card-type.svg)](https://travis-ci.org/braintree/credit-card-type) [![npm version](https://badge.fury.io/js/credit-card-type.svg)](http://badge.fury.io/js/credit-card-type)
+Credit Card Type [![Build Status](https://travis-ci.org/braintree/credit-card-type.svg)](https://travis-ci.org/braintree/credit-card-type) [![npm version](https://badge.fury.io/js/credit-card-type.svg)](http://badge.fury.io/js/credit-card-type) [![Bower](https://badge.fury.io/bo/credit-card-type.svg)](http://badge.fury.io/bo/credit-card-type)
 ================
 
 Credit Card Type provides a useful utility method for determining a credit card type from both fully qualified and partial numbers. This is not a validation library but rather a smaller component to help you build your own validation or UI library.
 
 This library is designed for type-as-you-go detection (supports partial numbers) and is written in CommonJS so you can use it in Node, io.js, and the [browser](http://browserify.org).
 
+## Download
+You can install credit-card-type via NPM or Bower.
```
Comment (jeffcarp) [inline README.md]: I'm not sure if this line is necessary since npm and Bower are listed directly below

File: README.md [hunk 2]
```diff
@@ -1,10 +1,25 @@
...
 ================
 
 Credit Card Type provides a useful utility method for determining a credit card type from both fully qualified and partial numbers. This is not a validation library but rather a smaller component to help you build your own validation or UI library.
 
 This library is designed for type-as-you-go detection (supports partial numbers) and is written in CommonJS so you can use it in Node, io.js, and the [browser](http://browserify.org).
 
+## Download
+You can install credit-card-type via NPM or Bower. 
+
+To install via NPM:
```
Comment (jeffcarp) [inline README.md]: s/NPM/npm/

**Comment by Henryvw** [comment]
@jeffcarp I made the recommended updates to the text. Cheers

File: README.md [hunk 3]
```diff
@@ -1,10 +1,23 @@
-Credit Card Type [![Build Status](https://travis-ci.org/braintree/credit-card-type.svg)](https://travis-ci.org/braintree/credit-card-type) [![npm version](https://badge.fury.io/js/credit-card-type.svg)](http://badge.fury.io/js/credit-card-type)
+Credit Card Type [![Build Status](https://travis-ci.org/braintree/credit-card-type.svg)](https://travis-ci.org/braintree/credit-card-type) [![npm version](https://badge.fury.io/js/credit-card-type.svg)](http://badge.fury.io/js/credit-card-type) [![Bower](https://badge.fury.io/bo/credit-card-type.svg)](http://badge.fury.io/bo/credit-card-type)
 ================
 
 Credit Card Type provides a useful utility method for determining a credit card type from both fully qualified and partial numbers. This is not a validation library but rather a smaller component to help you build your own validation or UI library.
 
 This library is designed for type-as-you-go detection (supports partial numbers) and is written in CommonJS so you can use it in Node, io.js, and the [browser](http://browserify.org).
 
+## Download
+To install via s/NPM/npm:
```
Comment (jeffcarp) [inline README.md]: Oops sorry for the confusion, I meant swap 'NPM' out with 'npm' (I think it's [sed syntax](http://www.gnu.org/software/sed/manual/html_node/The-_0022s_0022-Command.html)?)

If you could change it to `To install via npm:` (and also potentially squash all the commits into one commit) then I think we're good to merge! :smile:

**Comment by Henryvw** [comment]
:cactus: :+1:

**Comment by jeffcarp** [comment]
Merged! Thanks again @Henryvw! :fireworks: :stars:

**Comment by EvanHahn** [comment]
@Henryvw Yeah, thanks so much!

**Comment by Henryvw** [comment]
:dancers: cheers you're welcome!

**Pull request merged**